### DISCUSSION TITLE
default_double_support_static_ratio

### DISCUSSION THREAD

**Discussion by Naoki-Hiraoka** [YutaKojio/hrpsys-base#47] [Q&A] [answered]
`default_double_support_static_ratio`が0以外の値をとることには対応していないですか。また、`default_double_support_static_ratio`に0以外の値をセットしたくなるのはどんなときですか.

https://github.com/YutaKojio/hrpsys-base/blob/8e725bbe0a97a5a281cd2cf67803cff75a3244ea/rtc/AutoBalancer/GaitGenerator.cpp#L1327-L1356

**Comment by YutaKojio** [comment, marked as answer]
昔からあるパラメータで僕は使いたくなることがなかったので，僕の実装では使うことを考慮していない実装になっているかもです．
https://github.com/fkanehiro/hrpsys-base/blob/master/rtc/AutoBalancer/hrpsys_AutoBalancer_GaitGenerator_memo.pptx
